### ISSUE TITLE
Can I enque jobs from the database, e.g. in triggers?

### ISSUE THREAD

**Issue by TimoStolz** [que-rb/que#289]
Can I enque jobs from the database. I'd like to run jobs after insertions to specific tables. Of course, I could use rails hooks for this. However, the application has a lot of (micro-)services interacting with the database. It would be lovely, if all of these processes could dispatch jobs.

**Comment by hlascelles** [comment]
Agree, this would be a useful addition. I think the answer is "yes - we just need a good example of the code to do it".

**Comment by ZimbiX** [comment]
@TimoStolz if you figured this out, please let us know so we can add it to the documentation =)
